src/sentry/relay/projectconfig_cache/redis.py

```diff
@@ -29,16 +29,15 @@ def set_many(self, configs):
         p.execute()
 
     def delete_many(self, public_keys):
-        metrics.incr(
-            "relay.projectconfig_cache.write", amount=len(public_keys), tags={"action": "delete"}
-        )
-
         # Note: Those are multiple pipelines, one per cluster node
-        p = self.cluster.pipeline()
-        for public_key in public_keys:
-            p.delete(self.__get_redis_key(public_key))
+        with self.cluster.pipeline() as p:
+            for public_key in public_keys:
+                p.delete(self.__get_redis_key(public_key))
+            return_values = p.execute()
 
-        p.execute()
+        metrics.incr(
+            "relay.projectconfig_cache.write", amount=sum(return_values), tags={"action": "delete"}
+        )
```

Comment:
`sample_rate=1` for these?

Reply:
it wasn't before either.  generally i think the logic for sample_rate=1 is for when a metric only gets emitted rather rarely, but this is emitted a lot so i don't think this is needed.